wimprates/__init__.py

```diff
@@ -1,9 +1,18 @@
 __version__ = '0.4.1'
 
+from packaging import version
+if version.parse('0.4.1') < version.parse(__version__) < version.parse('0.6.0'):
+    # Remove this warning at some point
+    import warnings
+    warnings.warn(
+        'Default WIMP parameters are changed in accordance with '
+        'https://arxiv.org/abs/2105.00599 (github.com/JelleAalbers/wimprates/pull/14)')
```

Comment:
@JelleAalbers let me know if this is an acceptable level of verbosity?

Reply:
Sounds good! People can always use warning filters to silence this they are annoyed by it, but I think wimprates is used in enough legacy code that breaking backwards compatibility is worth something loud.